Applications/WebPortal/WebApp/library/source/pages/group_page/group_info_page/group_info_page.tsx

```diff
@@ -15,38 +15,26 @@ interface Properties {
 export class GroupInfoPage extends React.Component<Properties> {
 
   public render(): JSX.Element {
-    const {displaySize, group} = this.props;
     const membersCount = (() => {
-      if(displaySize !== DisplaySize.SMALL) {
+      if(this.props.displaySize !== DisplaySize.SMALL) {
         return (
           <span style={GroupInfoPage.STYLE.membersInfo}>
-            Total members: {group.length}
+            Total members: {this.props.group.length}
           </span>);
       } else {
         return null;
       }
     })();
-    const pageWidth = (() => {
-      if(displaySize === DisplaySize.SMALL) {
-        return GroupInfoPage.STYLE.smallPageWidth;
-      } else if(displaySize === DisplaySize.MEDIUM) {
-        return GroupInfoPage.STYLE.mediumPageWidth;
-      } else {
-        return GroupInfoPage.STYLE.largePageWidth; 
-      }
-    })();
+    const pageWidth = GroupInfoPage.STYLE[this.props.displaySize];
     const content = (() => {
-      if(group.length === 0) {
+      if(this.props.group.length === 0) {
         return (
           <div style={GroupInfoPage.STYLE.noEntries}>
             {GroupInfoPage.GROUP_EMPTY_MESSAGE}
           </div>);
       } else {
-        const entries = [];
-        for(const account of group) {
-          entries.push(<GroupMemberEntry account={account}/>);
-        }
-        return entries;
+        return this.props.group.map((account: AccountEntry) => 
```

Comment:
What happens if you remove the ```AccountEntry```?

Reply:
Nothing. I was having trouble compling for a moment which is why I added it in.

Reply:
Good good, then you can simplify to ```account => ...``` removing the type and the brackets.